.pre-commit-config.yaml

```diff
@@ -29,7 +29,4 @@ repos:
   - repo: https://github.com/psf/black
     rev: 24.10.0
     hooks:
-      - id: black
-        additional_dependencies: ['.[jupyter]']
-        types: [python, jupyter]
-        types_or: [python, jupyter]
+      - id: black-jupyter
```

Comment:
Why not use `black`?

Reply:
I think this PR is optional because our CI now automatically runs lint, which includes `black`, so all Python code is currently formatted with black and does not need to follow editor config.

Reply:
If I understand correctly, the previous hook should run on `.ipynb` files as well.

Reply:
> our CI now automatically runs lint, which includes black, so all Python code is currently formatted with black

This is not true. No Python file in this repo is covered in CI. See my comment https://github.com/sgl-project/sglang/pull/1926#issuecomment-2459066525. Also, you can verify it in the file changes in this PR. If all files are under the `black` style, there should be no Python files present in the changes.

Reply:
ref https://github.com/sgl-project/sglang/blob/a5e0defb5a560a6d42882008c1dd8a739002ab7d/.pre-commit-config.yaml#L32

Reply:
@zhaochenyang20 https://github.com/sgl-project/sglang/pull/1810/files
In this PR, the `black` rule is broken. Please fix it.